packages/coinjoin/src/client/round/connectionConfirmation.ts

```diff
@@ -176,5 +178,7 @@ export const connectionConfirmation = async (
             }),
     );
 
+    round.setSessionPhase(SessionPhase.AwaitingParticipants);
```

Comment:
this should be more like  "AwaitingOthersConfirmation"

`AwaitingOthersParticipants` shoud be in "inputRegistration" phase (After successful registration)

Reply:
15d9d0f14

Reply:
15d9d0f14